src/go/configgenerator/route_generator.go

```diff
@@ -403,6 +403,7 @@ func makeRoute(routeMatcher *routepb.RouteMatch, method *configinfo.MethodInfo)
 	}
 
 	return &routepb.Route{
```

Comment:
There are many other places where a route is constructed in this file. CORS, etc. Do control-f for `&routepb.Route{` just to see a few.

Reply:
Looks like the cors route doesn't associate with an method info? Is that so? In this case, what should be the route name?

Reply:
Ok, that's fine then.